### PR TITLE
Adds Stakeholder Input Process

### PR DESCRIPTION
**Pull request by tobie**
Relates to #64.

### CODE REVIEW

File: PROCESS.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 This document captures some of the administrative processes used to manage the AC in accordance with its [working mode](./WORKING_MODE.md).
 
+
+## Stakeholder Input Process
+
+Steps to take when an AC work item has been identified:
+
+- [ ] New issue is raised by stakeholder
+- [ ] Brought to the AC for discussion
```
Comment (tobie) [inline PROCESS.md:11]:
```suggestion
- [ ] Brought to the AC for discussion
- [ ] The AC identifies clear desired outcomes for the issue
```

File: PROCESS.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+- [ ] Issue is assigned to a champion (if no champion signs up, the issue is closed until a champion steps up)
+- [ ] Champion brings issue to the TSC and/or the relevant Working Groups (WG), either by filing an issue or joining a meeting
+- [ ] Champion drives the issue to resolution in collaboration with TSC/WG and stakeholder
+- [ ] Issue resolution is documented
```
Comment (tobie) [inline PROCESS.md:15]:
```suggestion
- [ ] Issue is brought back to the AC which recognizes that the desired outcomes has been reached, approves alternative outcomes, or suggests remediations
- [ ] Issue resolution is documented
```

**Comment by sumodas** [comment]
+1 

Looks good - thanks!

—
Sumantro Das
E: sumantro.das@gmail.com
P: (845) 558-8313

**Comment by gui-poa** [comment]
LGTM also!

**Pull request merged**